app/src/interfaces/input/input.vue

```diff
@@ -128,6 +128,7 @@ export default defineComponent({
 		const inputType = computed(() => {
 			if (props.masked) return 'new-password';
 			if (['bigInteger', 'integer', 'float', 'decimal'].includes(props.type)) return 'number';
+			if (['dateTime', 'date', 'time', 'timestamp'].includes(props.type)) return 'date';
```

Comment:
Previously this was a method, but that was before `inputType` computed value existed. Since now it's here, I moved the return `date` logic here based on the convention for return `number`.

Reply:
Does `<input type="date" />` work with all these value types?

Reply:
On hindsight, I'm now unsure when will this come to play? All date inputs are still technically 'text' atm right? Or is it for the input type="date" in future? I was just "translating" the code as-is so that's oversight on my part 😥

Reply:
> On hindsight, I'm now unsure when will this come to play? All date inputs are still technically 'text' atm right? Or is it for the input type="date" in future? I was just "translating" the code as-is so that's oversight on my part 😥

Yeah! That's what I was thinking too 🙂

Reply:
So can we say this is somewhat out of the scope of this PR? Perhaps we can shelve this as I assume #3728 can cover this? 🤔